tests/FakeItEasy.Specs/ConfiguringPropertySetterSpecs.cs

```diff
@@ -33,9 +32,11 @@ public interface IHaveInterestingProperties
             "When assignment of a property is configured using a null expression"
                 .x(() => exception = Record.Exception(() => A.CallToSet<int>(null)));
 
-            "Then the call to configure throws"
-                .x(() => exception.Should().BeAnExceptionOfType<ArgumentNullException>()
-                    .And.ParamName.Should().Be("propertySpecification"));
+            "Then an argument null exception is thrown"
+                .x(() => exception.Should().BeAnExceptionOfType<ArgumentNullException>());
+
+            "And the parameter name should be 'propertySpecification'"

```

Comment:
Didn't we decide earlier that step descriptions shouldn't contain "should" ? `"And the parameter name is 'propertySpecification'"`